components/scream/cime_config/config_component.xml

```diff
@@ -26,8 +26,9 @@
     <type>char</type>
     <valid_values></valid_values>
     <default_value></default_value>
-    <values modifier='additive'>
+    <values match='last'>
```

Comment:
@jgfouca is prob better than me to review XML stuff, but isn't this preventing people from appending CMake config options via xmlchange? Or can they still do `xmlchange --append SCREAM_CMAKE_OPTIONS="SOME_VAR my_value"`?

Reply:
Yes, it is for that purpose and make it work out of box.  This field currently uses only one long string as value, so 'last' is appropriate and high-res grid could match both entries.